### DISCUSSION TITLE
CTRL + V Problem

### DISCUSSION THREAD

**Discussion by Bisyk** [codewars/codewars.com#2981] [Q&A] [answered]
Hi guys, I have a problem with pasting and copying code in codewars environment. I use the Google Chrome browser, but the key combination ctrl+v or ctrl+c does not work at all, when using another browser, for example, Microsoft Edge, everything works correctly. What could be the reason and how to solve it? Thank you.

**Comment by Kacarott** [comment, marked as answer]
You may have accidentally activated the vim or emacs mode. Check in the top right corner just above the editor that one of those squares is not highlighted? If one is, just click it again to deactivate, and try pasting again.

**Reply by Bisyk** [reply]
Initially, they are both off, but turning one on and off fixes the problem. Thanks for the help.

**Comment by Ender-Coder76** [comment]
Can you Just Paste codes?

**Reply by Bisyk** [reply]
For some reason, sometimes only with the right mouse button. Try turning VIM on and off in the right corner. That might help.
